mbtiles/src/summary.rs

```diff
@@ -313,7 +313,7 @@ mod tests {
               - -123.75000000000001
               - -40.97989806962015
               - 180.00000000000003
-              - 61.60639637138628
+              - 61.60639637138627
```

Comment:
CI dooes not like this change.
Unsure why it comes up

```suggestion
              - 61.60639637138628
```